### PR TITLE
Add eslint rule: no floating promises

### PR DESCRIPTION
**Pull request by gimenete**
Also, I've fixed the `npm run lint` and `npm run lint:fix` npm scripts.

### CODE REVIEW

File: client/.eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  ignorePatterns: ["dist"],
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    project: 'client/tsconfig.json'
```
Comment (SferaDev) [inline client/.eslintrc.cjs:6]: Not sure this is taking the config file properly

File: codegen/.eslintrc.cjs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  ignorePatterns: ["dist", "example", "scripts"],
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    project: 'codegen/tsconfig.json'
```
Comment (SferaDev) [inline codegen/.eslintrc.cjs:6]: Same

File: client/.eslintrc.cjs (hunk 1 above)
Comment (gimenete) [reply client/.eslintrc.cjs:6]: 🤔 this is correct to run eslint from the root dir. Like in `eslint .`

How are you trying to run eslint? From the `client` dir in this case?

Comment (SferaDev) [reply client/.eslintrc.cjs:6]: Yup, as expected

Screenshot 2022-05-02 at 12 19 55

**Review by SferaDev — APPROVED** [review]

**Pull request merged**